androidApp/src/main/java/com/douglastaquary/busaoshare/android/ui/screens/search/viewModels/SearchTripViewModel.kt

```diff
@@ -16,10 +20,17 @@ class SearchTripViewModel(
 ) : ViewModel() {
 
     val tripListState = MutableStateFlow<UiState<List<Trip>>>(UiState.Empty)
-    private val _search = MutableStateFlow(null as String?)
-    val search: StateFlow<String?> = _search
     var isAuthenticated = MutableStateFlow<Boolean>(false)
 
+    private val _searchWidgetState: MutableState<SearchWidgetState> =
```

Comment:
Pelo que entendi aqui, esse MutableState e State tem o mesmo comportamento do MutableLiveData e LiveData no Android.
Se for isso, vai ser tranquilo de usar ele, porque já usava LiveData no MVVM.

Reply:
Boa! Eu confesso que ainda preciso estudar mais esses conceitos de state no Android. Pra saber de fato o que cada um tem de específico.

Reply:
A boa, demoro. Bora ir pesquinsando e trocando figurinha